Ground_RealSense_Height: 1.05
Ground_Neck_Height: 0.95
Ground_Arm_Height: 0.78
Shoulder_Elbow_Length: 0.22
Elbow_Wrist_Length: 0.17
Wrist_Endeffector_Length: 0.17

Origin_Angle: [980, 3040, 2065, 2065, 2150, 1794]
Gear_Ratio: [2.1, 2.1, 1, 1, 1, 1]
